### PR TITLE
Add new blog post about the CORE SIG

### PR DESCRIPTION
**Pull request by anajankovic**
**Description**

- Add new blog post about the CORE SIG

### CODE REVIEW

**Comment by magicmatatjahu** [comment]
Please remove `15-core-sig.md` file from root.

File: 15-core-sig.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+---
```
Comment (magicmatatjahu) [inline 15-core-sig.md]: Please remove this file

File: src/blog-posts/15-blog-post-core-sig.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
+---
+path: "/blog/core-sig"
+date: "2019-02-20"
+author: "Lukasz Gornicki, Product Owner @Kyma"
```
Comment (derberg) [inline src/blog-posts/15-blog-post-core-sig.md]: come one Ana, don't be shy, I'm not an author of this post and do not want to put my name on your great work

File: src/blog-posts/15-blog-post-core-sig.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+- Facilitate the collaboration with external contributors
+- Communicate the Kyma project vision and roadmap to the community
+- Represent the Kyma contributors
+- Be the main contribution contact for Kyma
+- Gauge interest, request the community feedback, and ensures it is addressed
+- Own all code repositories in the Kyma organization except for the community repository
+- Showcase proposals to the community and collects the feedback to those proposals to assist the decision-making process
+- Represent its members and the Kyma community in the Council meetings
+
+**I am interested! How can I join the CORE SIG?**
```
Comment (derberg) [inline src/blog-posts/15-blog-post-core-sig.md]: please also give link to slack and info about #sig-core channel

File: src/blog-posts/15-blog-post-core-sig.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+  - kyma, community, sig, core
+title: "The CORE Special Interest Group"
+---
+Behind the Kyma open source project there is a Kyma community who collaborates through Special Interest Groups (SIGs) and Working Groups (WGs), just like Kubernetes. In these groups, you can meet community leaders and members, exchange information on Kyma, provide feedback, raise interest, ask questions, and learn about Kyma news and updates. Additionally, both SIGs and WGs facilitate technical discussions, proposals, and contributions, and ensure Kyma is a welcoming community for all contributors. The WGs facilitate discussions and work on short-lived, concrete topics that either result from the work of SIG groups or which the community members initiate directly.
+
+**Is there a SIG I can join?**
+
+Yes, there is one main SIG called CORE as the entry point. The CORE SIG consists of SIG leaders and members. To learn more about the leaders and members role, click [here](https://github.com/kyma-project/community/blob/master/sig-and-wg/README.md#roles).
+
+**Who are the CORE SIG leaders?**
```
Comment (derberg) [inline src/blog-posts/15-blog-post-core-sig.md]: Please use twitter handles, that will ease communication and accessibility
https://twitter.com/PiotrasPk
https://twitter.com/derberq
https://twitter.com/Abd4llA
Mateusz is in progress of creating one, you can ping him to build preasure

File: src/blog-posts/15-blog-post-core-sig.md [hunk 5]
```diff
@@ -0,0 +1,40 @@
+---
+path: "/blog/core-sig
+date: "2019-02-20"
+author: "Ana Yankovich, Knowledge Consultant @Kyma"
+tags:
+  - kyma, community, sig, core
+title: "The CORE Special Interest Group"
```
Comment (derberg) [inline src/blog-posts/15-blog-post-core-sig.md]: I'm not sure you can now capitalize `CORE` here in the post everywhere, it should be `Core`, but let's see what TWs will say during review

File: src/blog-posts/15-blog-post-core-sig.md (hunk 4 above)
Comment (derberg) [reply src/blog-posts/15-blog-post-core-sig.md]: https://twitter.com/m_szostok

File: src/blog-posts/15-blog-post-core-sig.md [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+
+**Is there a SIG I can join?**
+
+Yes, there is one main SIG called CORE as the entry point. The CORE SIG consists of SIG leaders and members. To learn more about the leaders and members role, click [here](https://github.com/kyma-project/community/blob/master/sig-and-wg/README.md#roles).
+
+**Who are the CORE SIG leaders?**
+
+The CORE SIG leaders are Piotr Kopczynski [(@PK85](https://twitter.com/PiotrasPk)), Mateusz Szostok [(@mszostok](https://twitter.com/m_szostok)), Ahmed Abdalla ([@Abd4llA](https://twitter.com/Abd4llA)) and Lukasz Gornicki ([@derberg](https://twitter.com/derberq)).
```
Comment (mszostok) [inline src/blog-posts/15-blog-post-core-sig.md]:
```suggestion
The CORE SIG leaders are Piotr Kopczynski ([@PK85](https://twitter.com/PiotrasPk)), Mateusz Szostok [(@mszostok](https://twitter.com/m_szostok)), Ahmed Abdalla ([@Abd4llA](https://twitter.com/Abd4llA)) and Lukasz Gornicki ([@derberg](https://twitter.com/derberq)).
```

File: src/blog-posts/15-blog-post-core-sig.md [hunk 7]
```diff
@@ -0,0 +1,40 @@
+---
+path: "/blog/core-sig
```
Comment (magicmatatjahu) [inline src/blog-posts/15-blog-post-core-sig.md]: You forgot about closing the string 😈

File: src/blog-posts/15-blog-post-core-sig.md [hunk 8]
```diff
@@ -0,0 +1,40 @@
+---
+path: "/blog/core-sig"
+date: "2019-02-20"
+author: "Ana Yankovich, Knowledge Consultant @Kyma"
+tags:
+  - kyma, community, sig, core
+title: "The CORE Special Interest Group"
+---
+Behind the Kyma open source project there is a Kyma community who collaborates through Special Interest Groups (SIGs) and Working Groups (WGs), just like Kubernetes. In these groups, you can meet community leaders and members, exchange information on Kyma, provide feedback, raise interest, ask questions, and learn about Kyma news and updates. Additionally, both SIGs and WGs facilitate technical discussions, proposals, and contributions, and ensure Kyma is a welcoming community for all contributors. The WGs facilitate discussions and work on short-lived, concrete topics that either result from the work of SIG groups or which the community members initiate directly.
```
Comment (kazydek) [inline src/blog-posts/15-blog-post-core-sig.md]:
```suggestion
As you know, a vibrant community of individuals stands behind the Kyma open-source project. They collaborate through Special Interest Groups (SIGs) and Working Groups (WGs), just like it is done in Kubernetes. Additionally, both SIGs and WGs facilitate technical discussions, proposals, and contributions. They also ensure Kyma is a welcoming community for all contributors. The WGs facilitate discussions and work on short-lived, specific topics that either result from the work of SIG groups or that the community members initiate directly.
```

File: src/blog-posts/15-blog-post-core-sig.md [hunk 9]
```diff
@@ -0,0 +1,40 @@
...
+
+**Is there a SIG I can join?**
+
+Yes, there is one main SIG called CORE as the entry point. The CORE SIG consists of SIG leaders and members. To learn more about the leaders and members role, click [here](https://github.com/kyma-project/community/blob/master/sig-and-wg/README.md#roles).
```
Comment (kazydek) [inline src/blog-posts/15-blog-post-core-sig.md]:
```suggestion
Yes, there is one main SIG called **Core** that is the entry point. The Core SIG consists of SIG leaders and members. To learn more about their roles, click [here](https://github.com/kyma-project/community/blob/master/sig-and-wg/README.md#roles).
```

File: src/blog-posts/15-blog-post-core-sig.md [hunk 10]
```diff
@@ -0,0 +1,40 @@
...
+
+**Who are the CORE SIG leaders?**
```
Comment (kazydek) [inline src/blog-posts/15-blog-post-core-sig.md]:
```suggestion
**Who are the Core SIG leaders?**
```

File: src/blog-posts/15-blog-post-core-sig.md [hunk 11]
```diff
@@ -0,0 +1,40 @@
...
+
+The CORE SIG leaders are Piotr Kopczynski ([@PK85](https://twitter.com/PiotrasPk)), Mateusz Szostok [(@mszostok](https://twitter.com/m_szostok)), Ahmed Abdalla ([@Abd4llA](https://twitter.com/Abd4llA)) and Lukasz Gornicki ([@derberg](https://twitter.com/derberq)).
```
Comment (kazydek) [inline src/blog-posts/15-blog-post-core-sig.md]:
```suggestion
The Core SIG leaders are Piotr Kopczynski ([@PK85](https://twitter.com/PiotrasPk)), Mateusz Szostok [(@mszostok](https://twitter.com/m_szostok)), Ahmed Abdalla ([@Abd4llA](https://twitter.com/Abd4llA)), and Lukasz Gornicki ([@derberg](https://twitter.com/derberq)).
```

File: src/blog-posts/15-blog-post-core-sig.md [hunk 12]
```diff
@@ -0,0 +1,40 @@
...
+
+**What is the purpose of the CORE SIG?**
```
Comment (kazydek) [inline src/blog-posts/15-blog-post-core-sig.md]:
```suggestion
**What is the purpose of the Core SIG?**
```

File: src/blog-posts/15-blog-post-core-sig.md [hunk 13]
```diff
@@ -0,0 +1,40 @@
...
+
+The purpose of the CORE SIG is to:
```
Comment (kazydek) [inline src/blog-posts/15-blog-post-core-sig.md]:
```suggestion
The purpose of the Core SIG is to:
```

File: src/blog-posts/15-blog-post-core-sig.md [hunk 14]
```diff
@@ -0,0 +1,40 @@
...
+- Facilitate the collaboration with external contributors
+- Communicate the Kyma project vision and roadmap to the community
+- Represent the Kyma contributors
+- Be the main contribution contact for Kyma
+- Gauge interest, request the community feedback, and ensures it is addressed
+- Own all code repositories in the Kyma organization except for the community repository
+- Showcase proposals to the community and collects the feedback to those proposals to assist the decision-making process
+- Represent its members and the Kyma community in the Council meetings
+
+**I am interested! How can I join the CORE SIG?**
```
Comment (kazydek) [inline src/blog-posts/15-blog-post-core-sig.md]:
```suggestion
**I am interested! How can I join the Core SIG?**
```

File: src/blog-posts/15-blog-post-core-sig.md [hunk 15]
```diff
@@ -0,0 +1,40 @@
...
+
+The purpose of the CORE SIG is to:
+
+- Publicly represent Kyma
+- Facilitate the collaboration with external contributors
+- Communicate the Kyma project vision and roadmap to the community
+- Represent the Kyma contributors
+- Be the main contribution contact for Kyma
+- Gauge interest, request the community feedback, and ensures it is addressed
+- Own all code repositories in the Kyma organization except for the community repository
```
Comment (kazydek) [inline src/blog-posts/15-blog-post-core-sig.md]:
```suggestion
- Own all code repositories in the Kyma organization except for the `community` repository.
```

File: src/blog-posts/15-blog-post-core-sig.md [hunk 16]
```diff
@@ -0,0 +1,40 @@
...
+- Showcase proposals to the community and collects the feedback to those proposals to assist the decision-making process
```
Comment (kazydek) [inline src/blog-posts/15-blog-post-core-sig.md]:
```suggestion
- Showcase proposals to the community and collect feedback on those proposals to assist the decision-making process.
```

File: src/blog-posts/15-blog-post-core-sig.md [hunk 17]
```diff
@@ -0,0 +1,40 @@
...
+- Represent its members and the Kyma community in the Council meetings
+
+**I am interested! How can I join the CORE SIG?**
+
+You can join the CORE SIG biweekly meeting that takes place on Wednesdays at 15:00 CET on [Zoom](https://zoom.us/j/4794339038). You can also read the meeting agenda and notes [here](https://docs.google.com/document/d/1vWleTon7sJIk0teee4SoVezS4mR3K8TlkvXkgLJwRD8). You can also watch the past SIG CORE meetings on the [Kyma YouTube channel](https://www.youtube.com/watch?v=1DD5aCBnywQ&amp;list=PL7PGl--iaIH9jnbT8yG_KOV3unv1iUILR).
```
Comment (kazydek) [inline src/blog-posts/15-blog-post-core-sig.md]:
```suggestion
You can join the Core SIG biweekly meetings that take place on Wednesdays at 15:00 CET on [Zoom](https://zoom.us/j/4794339038). If you can't join them, read the meeting agenda and notes [here](https://docs.google.com/document/d/1vWleTon7sJIk0teee4SoVezS4mR3K8TlkvXkgLJwRD8), or watch the meeting recordings on the [Kyma YouTube channel](https://www.youtube.com/watch?v=1DD5aCBnywQ&amp;list=PL7PGl--iaIH9jnbT8yG_KOV3unv1iUILR).
```

File: src/blog-posts/15-blog-post-core-sig.md [hunk 18]
```diff
@@ -0,0 +1,40 @@
...
+
+We're on [Slack](https://join.slack.com/t/kyma-community/shared_invite/enQtNDAwNzE4Mjk2NDE3LTJhOTlmZjM5YzkwNmEzNmY3ZjE2MTU2OTMxOGE4ZDM0MmU4ZWRkZGJiODgzNmRmMTYxMDYwNjZiMDAwMTA2OWM) too!
+Join the [#sig-core Slack channel](https://kyma-community.slack.com/messages/CBP7LKRPS/) to ask questions, keep up to date with the latest SIG CORE news or simply to meet CORE SIG leaders and members.
```
Comment (kazydek) [inline src/blog-posts/15-blog-post-core-sig.md]:
```suggestion
Join the [#sig-core Slack channel](https://kyma-community.slack.com/messages/CBP7LKRPS/) to ask questions, keep up to date with the latest Core SIG news, or simply to meet the Core SIG leaders and members. 
```

File: src/blog-posts/15-blog-post-core-sig.md [hunk 19]
```diff
@@ -0,0 +1,40 @@
...
+- Represent its members and the Kyma community in the Council meetings
+
+**I am interested! How can I join the CORE SIG?**
+
+You can join the CORE SIG biweekly meeting that takes place on Wednesdays at 15:00 CET on [Zoom](https://zoom.us/j/4794339038). You can also read the meeting agenda and notes [here](https://docs.google.com/document/d/1vWleTon7sJIk0teee4SoVezS4mR3K8TlkvXkgLJwRD8). You can also watch the past SIG CORE meetings on the [Kyma YouTube channel](https://www.youtube.com/watch?v=1DD5aCBnywQ&amp;list=PL7PGl--iaIH9jnbT8yG_KOV3unv1iUILR).
+
+We're on [Slack](https://join.slack.com/t/kyma-community/shared_invite/enQtNDAwNzE4Mjk2NDE3LTJhOTlmZjM5YzkwNmEzNmY3ZjE2MTU2OTMxOGE4ZDM0MmU4ZWRkZGJiODgzNmRmMTYxMDYwNjZiMDAwMTA2OWM) too!
+Join the [#sig-core Slack channel](https://kyma-community.slack.com/messages/CBP7LKRPS/) to ask questions, keep up to date with the latest SIG CORE news or simply to meet CORE SIG leaders and members. 
+
+We're happy to have you at the CORE SIG!
```
Comment (kazydek) [inline src/blog-posts/15-blog-post-core-sig.md]:
```suggestion
We'd be happy to have you at the Core SIG!
```

File: src/blog-posts/15-blog-post-core-sig.md [hunk 20]
```diff
@@ -0,0 +1,40 @@
...
+
+**Who are the CORE SIG leaders?**
+
+The CORE SIG leaders are Piotr Kopczynski ([@PK85](https://twitter.com/PiotrasPk)), Mateusz Szostok [(@mszostok](https://twitter.com/m_szostok)), Ahmed Abdalla ([@Abd4llA](https://twitter.com/Abd4llA)) and Lukasz Gornicki ([@derberg](https://twitter.com/derberq)).
+
+**What is the purpose of the CORE SIG?**
+
+The purpose of the CORE SIG is to:
+
+- Publicly represent Kyma
```
Comment (kazydek) [inline src/blog-posts/15-blog-post-core-sig.md]:
```suggestion
- Publicly represent Kyma.
```

File: src/blog-posts/15-blog-post-core-sig.md [hunk 21]
```diff
@@ -0,0 +1,40 @@
...
+- Facilitate the collaboration with external contributors
```
Comment (kazydek) [inline src/blog-posts/15-blog-post-core-sig.md]:
```suggestion
- Facilitate collaboration with external contributors.
```

File: src/blog-posts/15-blog-post-core-sig.md [hunk 22]
```diff
@@ -0,0 +1,40 @@
...
+- Communicate the Kyma project vision and roadmap to the community
```
Comment (kazydek) [inline src/blog-posts/15-blog-post-core-sig.md]:
```suggestion
- Communicate the Kyma project vision and roadmap to the community.
```

File: src/blog-posts/15-blog-post-core-sig.md [hunk 23]
```diff
@@ -0,0 +1,40 @@
...
+- Represent the Kyma contributors
```
Comment (kazydek) [inline src/blog-posts/15-blog-post-core-sig.md]:
```suggestion
- Represent the Kyma contributors.
```

File: src/blog-posts/15-blog-post-core-sig.md [hunk 24]
```diff
@@ -0,0 +1,40 @@
...
+- Be the main contribution contact for Kyma
```
Comment (kazydek) [inline src/blog-posts/15-blog-post-core-sig.md]:
```suggestion
- Be the main contribution contact for Kyma.
```

File: src/blog-posts/15-blog-post-core-sig.md [hunk 25]
```diff
@@ -0,0 +1,40 @@
...
+- Gauge interest, request the community feedback, and ensures it is addressed
```
Comment (kazydek) [inline src/blog-posts/15-blog-post-core-sig.md]:
```suggestion
- Gauge interest, request the community feedback, and ensure it is addressed.
```

File: src/blog-posts/15-blog-post-core-sig.md [hunk 26]
```diff
@@ -0,0 +1,40 @@
...
+- Own all code repositories in the Kyma organization except for the community repository
+- Showcase proposals to the community and collects the feedback to those proposals to assist the decision-making process
+- Represent its members and the Kyma community in the Council meetings
```
Comment (kazydek) [inline src/blog-posts/15-blog-post-core-sig.md]:
```suggestion
- Represent its members and the Kyma community in the Council meetings.
```

**Pull request merged**